tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -897,7 +897,6 @@ public function callAPISuccessGetValue($entity, $params, $type = NULL) {
   public function callAPISuccessGetSingle($entity, $params, $checkAgainst = NULL) {
     $params += array(
       'version' => $this->_apiversion,
-      'debug' => 1,
```

Comment:
@eileenmcnaughton in fact ```getAndCheck``` doesn't pass this parameter ```debug``` as in [here](https://github.com/civicrm/civicrm-core/blob/master/tests/phpunit/CiviTest/CiviUnitTestCase.php#L2606) and for API's result array comparison, this extraneous information are avoided while checking:

```
  public function assertAPIArrayComparison($result, $expected, $valuesToExclude = array(), $prefix = '') {
    $valuesToExclude = array_merge($valuesToExclude, array('debug', 'xdebug', 'sequential'));
...
```